.jenkins/perf_compile

```diff
@@ -39,7 +39,7 @@ pipeline {
     string(name: 'db_user', defaultValue: "${PIPELINE_USER}", description: 'Username for the databse')
     string(name: 'db_password', defaultValue: "${PIPELINE_PWD}", description: 'Password for the user')
     string(name: 'docker_registry', defaultValue: "${headnode}:5000", description: 'Name of the docker registry for pushing images')
-    string(name: 'base_image', defaultValue: '', description: 'Put a fully qualified docker name here to use (optional)')
+    //string(name: 'base_image', defaultValue: '', description: 'Put a fully qualified docker name here to use (optional)')
```

Comment:
comment? I see lots of comments left here, is there a reason for leaving them im?

Reply:
removed them